lib/render/leaderboard.php

```diff
@@ -361,14 +363,11 @@ function RenderTopAchieversComponent($user, array $gameTopAchievers, array $game
  *            4 - All Time
  * @param int $sort Stats to sort by
  *            1 - User
- *            2 - Total Achievement
- *            3 - Softcore Achievements
- *            4 - Hardcore Achievements
- *            5 - Points
- *            6 - Retro Points
- *            7 - Retro Ratio
- *            8 - Completed Awards
- *            9 - Mastered Awards
+ *            2 - Hardcore Achievements
+ *            3 - Hardcore Points
+ *            4 - Retro Points
+ *            5 - Retro Ratio
+ *            6 - Mastered Awards
```

Comment:
Changing these values could break bookmarks (not that I expect many people to bookmark this page). It might be best to just leave them as unsupported:
```
 *            1 - User
 *            2 - Total Achievements (no longer supported)
 *            3 - Softcore Achievements (no longer supported)
 *            4 - Hardcore Achievements
 *            5 - Hardcore Points
 *            6 - Retro Points
 *            7 - Retro Ratio
 *            8 - Completed Awards (no longer supported)
 *            9 - Mastered Awards
```

This is reflected by the fact that the userInfo page has a hardcoded `s=5`:
https://github.com/RetroAchievements/RAWeb/blob/f366b690c3a06565b6b1d1900c644edc0d6b2e0d/public/userInfo.php#L268
which should probably just be removed as it reflects the default sort

Reply:
Hmm not too concerned about the bookmarks issue but I changed the values back and noted which ones are unsupported like you mentioned. I think the userInfo link is still relevant since it includes `$rankOffset` to take you to the correct page for the user so I'll keep that for now.